server/__tests__/integration/capture/capture.spec.js

```diff
@@ -18,24 +22,34 @@ describe('/captures', () => {
 
   const modCapture = {
     ...capture2,
-    attributes: { entries: capture2.attributes },
+    attributes: { entries: attributes.attributes }
   };
 
   const updatedModCapture = { ...modCapture, ...captureUpdates };
   before(async () => {
+    await knex('grower_account').insert({
+      ...grower_account1,
+      status: 'active',
+    });
     await knex('grower_account').insert({
       ...grower_account2,
       status: 'active',
     });
   });
 
+
   after(async () => {
+
+    await knex('capture_tag')
+      .del();
+
+    await knex('tag')
+      .del();
+  
     await knex('capture')
-      .where({ ...updatedModCapture })
```

Comment:
My thinking with this "specific deleting" was to "make sure" that the updates made to the resource were persisted

Reply:
All checks should happen within the test itself.  The after() function is just for cleaning up.  Nothing in after() should be dependent on what happens in the tests themselves, or what is tested.